pyrs/transpiler.py

```diff
@@ -120,15 +121,15 @@ def usings(self):
         #![allow(unused_mut)]
         #![allow(unused_parens)]
         """
-        )
+        ) if not self._no_prologue else ""
         cargo_toml = f"""
         //! ```cargo
         //! [package]
         //! edition = "2018"
         //! [dependencies]
         //! {deps_str}
         //! ```
-        """
+        """ if not self._no_prologue else ""
```

Comment:
IMO, this should go back to using `cargo-deps` which are shorter, and be preceeded with a one liner comment to run the script using `cargo-eval` (and we should switch to cargo-eval immediately if we go down this path).

`edition = "2018"` can be supplied as a CLI arg - it is a constant, and doesnt need to vary per script.

That way the reader can quickly see which libraries are being used, and can easily see how to invoke it.

Reply:
Agree that cargo-deps was shorter and preferable. I don't have a strong opinion on cargo script vs eval.
Whatever we choose, it'd be nice if it also works on rust playground. The web UI links to it now.